docs/UpgradeGuide.md

```diff
@@ -110,38 +105,73 @@ Datadog.configure do |c|
 end
 ```
 
-Check out our [complete list of configuration changes](#appendix-configuration) for more details.
+Check out the table below for a list of common mappings:
```

Comment:
It isn't a full list, its an abbreviated list. And the implication behind common is correct; it may be missing some mappings, which we excluded for brevity (e.g. `tracing.distributed_tracing` vs `tracing.distributed_tracing.propagation_extract_style` and `tracing.distributed_tracing.propagation_inject_style`)